### PR TITLE
[ARCHETYPE-652] Added system requirements history

### PR DESCRIPTION
**Pull request by BrowneMonke**
1. Added info not visible on website
2. Required java version not found for some archetype versions

### CODE REVIEW

File: maven-archetype-plugin/pom.xml [hunk 1]
```diff
@@ -172,6 +172,154 @@
...
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-plugin-report-plugin</artifactId>
+          <configuration>
+
+            <requirementsHistories>
+
+              <requirementsHistory>
```
Comment (elharo) [inline maven-archetype-plugin/pom.xml:184]: let's drop the 1.0 and 2.0 alpha versions.

**Review by slachiewicz — APPROVED** [review]

File: maven-archetype-plugin/pom.xml (hunk 1 above)
Comment (slachiewicz) [reply maven-archetype-plugin/pom.xml:184]: i'll fix it

**Comment by slachiewicz** [comment]
Thank You @BrowneMonke for the initial PR. 
I've updated it to match the configuration for the latest report plugin version and merged it with 2b58aebfbe2398dfc1374114341ec47a2a63c8e4 
It also includes now planned 3.3.0

**Comment by jira-importer** [comment]
Resolve #871

**Comment by jira-importer** [comment]
Resolve #601

**Pull request closed without merging**